examples/Multi_Armed_Bandit/bernoulli_multi_armed_bandit.py

```diff
@@ -0,0 +1,175 @@
+    """Win rates associated with the arms."""
+
+    def draw_arm(self, arm_index: int) -> bool:
+        """Draw a reward by pulling the specified arm."""
+        return bernoulli(self.win_rates[arm_index]).rvs()
+
+
+### Setup
+
+# For our example, we use a system with a fixed collection of win rates, which we
+# assume are unknown to us:
+
+WIN_RATES = [0.3, 0.4, 0.6, 0.65, 0.8]
+BANDIT = MultiArmedBandit(WIN_RATES)
+
+# Given this target system, we consider the following two tasks:
+# - **Online optimization:** Maximizing the total reward accumulated during a campaign
+# - **Active learning:** Estimating the win rates of all arms
+
+# Solving these task can be achieved through an appropriate choice of acquisition function:
+# For online optimization, we use {class}`~baybe.acquisition.acqfs.qThompsonSampling`.
+# For active learning, we switch to {class}`~baybe.acquisition.acqfs.PosteriorStandardDeviation`.
+
+# To estimate the corresponding effects, we simulate each campaign for a certain number of steps and repeat this process in multiple Monte Carlo runs:
+
+ACQFS = [
+    qThompsonSampling(),  # Online optimization
+    PosteriorStandardDeviation(),  # Active learning
+]
+N_MC_RUNS = 10
+N_ITERATIONS = 200
+
+
+### Building the Model
+
+# To model the rewards obtained from the bandit, we use a {class}`~baybe.targets.binary.BinaryTarget`.
+# The corresponding search space is spanned by a {class}`~baybe.parameters.categorical.CategoricalParameter` describing the bandit's arms.
+
+n_arms = len(WIN_RATES)
+objective = BinaryTarget(name="clicked").to_objective()
+searchspace = CategoricalParameter(
+    name="Bandit Arm", values=[f"arm_{i}" for i in range(n_arms)]
+).to_searchspace()
+surrogate = BernoulliMultiArmedBanditSurrogate()
+
+
+# For each simulation, we report the trajectory of earned rewards and the estimated win rates of the bandit arms:
+
+
+class SimulationResult(NamedTuple):
+    """Result of a bandit campaign simulation."""
+
+    earned_rewards: np.ndarray
+    estimated_win_rates: np.ndarray
+
+
+# Putting all pieces together, we define a function to simulate our campaign under a given acquisition strategy:
+
+
+def simulate(acqf: AcquisitionFunction) -> SimulationResult:
+    """Simulate the campaign with with the given acquisition function."""
+    recommender = TwoPhaseMetaRecommender(
+        initial_recommender=RandomRecommender(
+            allow_repeated_recommendations=True,
+            allow_recommending_already_measured=True,
+        ),
+        recommender=BotorchRecommender(
+            surrogate_model=surrogate,
+            acquisition_function=acqf,
+            allow_repeated_recommendations=True,
+            allow_recommending_already_measured=True,
+        ),
+    )
+
+    # Containers for storing the rewards and win rate estimates
+    earned_rewards = np.zeros((N_MC_RUNS, N_ITERATIONS))
+    estimated_win_rates = np.zeros((N_MC_RUNS, n_arms))
+
+    for mc in range(N_MC_RUNS):
```

Comment:
do we want to use tqdm to provide progress info for people running this??

Reply:
tqdm stuff is not part of the online documentation - this would hence only influence users that run the code locally, and I do not think that this is relevant for them.

Reply:
hmm I was refering to exactly that: users that run this lcoally, theys ee an empty console while its running -> not ideal
consider for isntance any sciprt that uses our simulate capabilities -> nice progress bar

Reply:
Would love to add it but messes with the rendered documentation. Thus, will skip it since running out of time. We can reconsider when adding the GIFs (same question for other examples, btw)